test/functional/p2p_compactblocks.py

```diff
@@ -387,7 +387,7 @@ def test_compactblock_requests(self, test_node):
 
             if announce == "inv":
                 test_node.send_message(msg_inv([CInv(MSG_BLOCK, block.sha256)]))
-                test_node.wait_until(lambda: "getheaders" in test_node.last_message, timeout=30)
+                test_node.wait_for_getheaders()
```

Comment:
```suggestion
                test_node.wait_for_getheaders(timeout=30)
```

Reply:
I didn't really see the point of the shorter timeout here, but fair, I guess it's better to leave it as is if this is simply a refactor